### DISCUSSION TITLE
Is there a roadmap on future milestones?

### DISCUSSION THREAD

**Discussion by actraiser** [Accenture/reactive-interaction-gateway#365] [General]
I was wondering if there is a roadmap or maybe even something like a public Trello board where users can follow the progress on the project or even get involved in helping out on planned features or pending bug fixes. 

Either way, it would be great to hear about future milestones, in particular on the 3.0.0 version. Right now I use 3.0.0-alpha1 but I think there was not much activity on this branch for some time.

greets
-act

**Comment by mmacai** [comment]
@actraiser Hi, we are using Github milestones as a roadmap - https://github.com/Accenture/reactive-interaction-gateway/milestone/9 (should be visible to everyone).

There was not much activity in past few weeks as we were quite occupied with different topics. However, should get better in next few weeks (I expect end of March) - at least for me - and will again start pushing remaining items to complete the release of the version 3.

**Reply by actraiser** [reply]
That sounds great! Thank you for letting us know Mario - and the pointer to the Github milestones.

greets
-act
